src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const centerX = (wall.offsetWidth - spider.offsetWidth) / 2;
+const centerY = (wall.offsetHeight - spider.offsetHeight) / 2;
```

Comment:
These calculations for the center of the wall are only done once, when the script is loaded. If the size of the wall or the spider changes after this (for example, if the user resizes the browser window), the spider will no longer be in the center. To handle this, you could add an event listener for the window's 'resize' event, and recalculate the center position inside that event handler.